.github/replies.yml

```diff
@@ -0,0 +1,5 @@
+replies:
+  - body: |
+      <img alt="Maury meme: You checked 'I have searched for related issues'. A cursory issue search related issues determined that was a lie." src="https://user-images.githubusercontent.com/3335181/186501430-7f4c481a-f557-48b0-826c-74bb8434f4d5.png" height="483" width="488">\
+      Thanks for posting, but this is a duplicate of #<insert issue>. Next time, please use the search 🙂.
```

Comment:
Is this too passive aggressive for general use? 🤔

Reply:
Yeah whilst I love memes - I think that the meme is a bit too passive-aggressive.